File: frontend/src/app/dto/DeleteSurveyResponse.ts

export class DeleteSurveyResponse {
  statusCode: number;
  message: String;
  rowsAffected: number

  constructor(statusCode, message, rowsAffected) {
    this.statusCode = statusCode;
    this.message = message;
    this.rowsAffected = rowsAffected;
  }
}

File: src/main/java/is/larsen/ebbi/Model/responses/DeleteSurveyResponse.java
package is.larsen.ebbi.Model.responses;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class DeleteSurveyResponse extends BaseResponse {
    private Integer rowsAffected;

    public DeleteSurveyResponse(Integer status, String message, Integer rowsAffected) {
        super(status, message);
        this.rowsAffected = rowsAffected;
    }
}
